flang/lib/Frontend/FrontendActions.cpp

```diff
@@ -713,10 +713,18 @@ void CodeGenAction::generateLLVMIR() {
   pm.addPass(std::make_unique<Fortran::lower::VerifierPass>());
   pm.enableVerifier(/*verifyPasses=*/true);
 
+  const auto targetOpts = ci.getInvocation().getTargetOpts();
+  const llvm::Triple triple(targetOpts.triple);
```

Comment:
Is this needed?

Reply:
Nope, it's from an unrelated change - should be moved out of this patch. [It is needed in the other change!]